ci/Jenkinsfiles/build.groovy

```diff
@@ -242,7 +323,7 @@ pipeline {
               // first substitute docker image names and versions
               sh """
                 cp values.yaml values.yaml.tosubst
-                DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME} envsubst < values.yaml.tosubst > values.yaml
+                DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME} DOCKER_WEBUI_IMAGE_NAME=${DOCKER_WEBUI_IMAGE_NAME} envsubst < values.yaml.tosubst > values.yaml
```

Comment:
not sure the variables here are needed after all: they should already be avail as env variables

Reply:
you're right, if we were to limit the vars to be replaces they'd need to be passed as input to `envsubst` iirc